src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/AcceleratorKeyConverter.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class AcceleratorKeyConverter : IValueConverter
```

Comment:
Is this something you copied from another repo? I did a quick search and couldn't find, so wondering if there's a common practice that avoids converters?

Reply:
nit: Also, I believe these are actually "Access Keys", not accelerators.

Reply:
Thanks for poiting out. The correct property should be AccessKey

It seems setting AutomationProperties.AccessKey has no effect on keyboard accelertor behaviour.

So, I'm going to remove converter.